boards/kyria/kb_v2.py

```diff
@@ -0,0 +1,67 @@
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.quickpin.pro_micro.sparkfun_promicro_rp2040 import pinout as pins
+from kmk.scanners import DiodeOrientation
+from kmk.scanners import intify_coordinate as ic
+from storage import getmount
+
+isLeftSide = False if str(getmount('/').label)[-1] == 'R' else True
```

Comment:
This only works if the split sides were set up to use the mountpoint name to determine left and right.
The universal and future-proof way would be to request the "split-side" of a board from the `split` module. That's not technically your responsibility. The split module doesn't yet offer that functionality as, for example, a utility function. It should though, and I'm using this opportunity to point that out.
I'm proposing to factor out the split side/location heuristic into a public auxiliary function within the split module. Then you can call that sometime during the initialization to figure out what column and row pins to use.